src/Microsoft.AspNet.OData.Shared/EnableQueryAttribute.cs

```diff
@@ -751,8 +751,8 @@ internal static void ValidateSelectExpandOnly(ODataQueryOptions queryOptions)
                 throw Error.InvalidOperation(SRResources.QueryGetModelMustNotReturnNull);
             }
 
-            IEdmEntityType baseEntityType = model.GetEdmType(elementClrType) as IEdmEntityType;
-            IEdmStructuredType structuredType = model.GetEdmType(elementClrType) as IEdmStructuredType;
+            IEdmEntityType baseEntityType     = model.GetTypeMappingCache().GetEdmType(elementClrType, model)?.Definition as IEdmEntityType;
+            IEdmStructuredType structuredType = model.GetTypeMappingCache().GetEdmType(elementClrType, model)?.Definition as IEdmStructuredType;
```

Comment:
>?. [](start = 109, length = 2)

I think the build used to fail for one of the older solutions when you used "?.". I guess that is no longer the case.